.github/workflows/qa.yaml

```diff
@@ -225,7 +225,7 @@ jobs:
           artifacts_dir=$(mktemp -d --tmpdir authd-test-artifacts-XXXXXX)
           echo AUTHD_TESTS_ARTIFACTS_PATH="${artifacts_dir}" >> $GITHUB_ENV
 
-          echo ASAN_OPTIONS="log_path=${artifacts_dir}/asan.log:print_stats=true" >> $GITHUB_ENV
+          echo ASAN_OPTIONS="log_path=${artifacts_dir}/asan.log:print_stats=true:symbolize=1" >> $GITHUB_ENV
```

Comment:
This shouldn't be necessary,the flag should default to `true`:
https://github.com/google/sanitizers/wiki/SanitizerCommonFlags

Reply:
Yeah, I was aware... But since we got some results now I'm reluctant on dropping it :joy:

Reply:
Well, if we remove it and the stack traces start not being symbolized again, we'll notice. So I would remove it now and merge the PR, and then, in the unlikely case that symbols are still missing, we can add it again in a follow-up PR.

Reply:
Ok droppped :)